packages/graphql/src/schema/get-where-fields.ts

```diff
@@ -60,8 +60,8 @@ function getWhereFields({
                     return res;
                 }
 
-                res[`${f.fieldName}_IN`] = `[${f.typeMeta.input.where.pretty}]`;
-                res[`${f.fieldName}_NOT_IN`] = `[${f.typeMeta.input.where.pretty}]`;
+                res[`${f.fieldName}_IN`] = `[${f.typeMeta.input.where.pretty}${f.typeMeta.required ? "!" : ""}]`;
```

Comment:
**suggestion (non-blocking)**:
A helper function `makeRequired(str: string, required: boolean): string` to make a string required could be helpful to improve readability and make this common use case a bit more consistent in our codebase, something a bit like:

```suggestion
                res[`${f.fieldName}_IN`] = `[${makeRequired(f.typeMeta.input.where.pretty, f.typeMeta.required) }]`;
```

Reply:
This use case is actually pretty uncommon - normally I would have just been able to use `f.typeMeta.input.where.pretty` but this new nullability logic for `_IN` means it needs to be slightly different here. So I think a helper would only be used in this one place anyway.